content/engineering-practice-feature-flags.md

```diff
@@ -0,0 +1,119 @@
+Title: Engineering Practice Feature Flags
+Date: 2025-02-21
+Category: Posts 
+Tags: code, code-quality, engineering, technical-excellence
+Slug: engineering-practice-feature-flags
+Author: Martin M. Lacey
+Summary: Feature Flags Part 1 - How To Keep Trunk-Based Branching Reliable
+
+# What are Feature Flags?
+
+
+> ![software bug-bear - feature flag misunderstandings](/images/engineering-practices-feature-flags-3.jpg)
```

Comment:
Image not showing: file is called engineering-practices-feature-flags-3.jpg - *-feature-* not *-features-*.